provisioning_templates/provision/kickstart_default.erb

```diff
@@ -44,7 +44,19 @@ This template accepts the following parameters:
 install
 <% end -%>
 <% if host_param('kickstart_liveimg') %>
-liveimg --url=<%= host_param('kickstart_liveimg') %> <%= proxy_string %>
+<%
+img_name = host_param('kickstart_liveimg')
```

Comment:
This variable could be inside the if - else block.

Is this the final version? I see two commits after @iNecas 's LGTM.

Reply:
Lemme try again, I was wondering if this is better:

```ruby
liveimg_url = if host_param('kt_activation_keys')
  repository_url(img_name, 'isos')
else
  img_name = host_param('kickstart_liveimg')
  if ! img_name.nil? && img_name.match(%r|^([\w\-\+]+)://|)
    img_name
  else
    "#{@host.operatingsystem.medium_uri(@host)}/#{img_name}"
  end
end
```

Reply:
`img_name` is used on L2 in your snippet as well. Apart from this, I don't plan on making any more changes to this PR

Reply:
Wait, where exactly is used? I don't see it being used, you introduced this variable in this PR.